### PR TITLE
Issue #1167 - Document orphaned resources, update proj CLI

### LINKED ISSUE #1167: Ability to generate a warn/alert when a namespace deviates from the expected state
It would be a useful security feature if we can warn/alert when a namespace deviates from the expected state. E.g. there is a pod that shouldn't be running based on the deployment manifests.

### PR DESCRIPTION
**Pull request by alexmt**
Closes #1167

### CODE REVIEW

File: cmd/argocd/commands/project.go [hunk 1]
```diff
@@ -516,7 +536,12 @@ func printProjectLine(w io.Writer, p *v1alpha1.AppProject) {
 	default:
 		namespaceBlacklist = fmt.Sprintf("%d resources", len(p.Spec.NamespaceResourceBlacklist))
 	}
-	fmt.Fprintf(w, "%s\t%s\t%v\t%v\t%v\t%v\n", p.Name, p.Spec.Description, destinations, sourceRepos, clusterWhitelist, namespaceBlacklist)
+	if p.Spec.OrphanedResources == nil {
```
Comment (alexec) [inline cmd/argocd/commands/project.go]: what about `func (o *OrphanedResources) String()`?

File: docs/user-guide/orphaned-resources.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Orphaned Resources Monitoring
```
Comment (alexec) [inline docs/user-guide/orphaned-resources.md]: I have committed some grammar changes to this

**Review by alexec — CHANGES_REQUESTED** [review]

File: cmd/argocd/commands/project.go (hunk 1 above)
Comment (alexmt) [reply cmd/argocd/commands/project.go]: This struct has autogenernated String method, so I cannot override it. I agree it is bad to duplicate this logic, so I've created `formatOrphanedResources` function.

File: docs/user-guide/orphaned-resources.md (hunk 2 above)
Comment (alexmt) [reply docs/user-guide/orphaned-resources.md]: thank you!

**Comment by alexmt** [comment]
@alexec , please take another look

**Review by alexec — APPROVED** [review]

**Pull request merged**